Formula/ldid.rb

```diff
@@ -18,7 +18,13 @@ class Ldid < Formula
   depends_on "openssl"
 
   def install
-    inreplace "./make.sh", %r{^.*/Applications/Xcode-5.1.1.app.*}, ""
+    inreplace "make.sh" do |s|
+      s.gsub! %r{^.*/Applications/Xcode-5.1.1.app.*}, ""
+
+      # Reported upstream 2 Sep 2018 (to saurik via email)
```

Comment:
@DomT4 Email sent, also carbon-copied you just FYI. 😉

Reply:
Thanks! LGTM :shipit: